.github/workflows/ci.yml

```diff
@@ -144,25 +144,30 @@ jobs:
           # Used to avoid rate limits when fetching the releases from k3s repo.
           # Anonymous access is limited to 60 requests / hour / worker
           # github-token: ${{ secrets.GITHUB_TOKEN }}
-          k3d-args: "--no-lb --k3s-arg --no-deploy=traefik,servicelb,metrics-server@server:*"
+          k3d-args: "--no-lb --no-rollback --k3s-arg --disable=traefik,servicelb,metrics-server@server:*"
+
+      # Set a hacky workaround evar for https://github.com/kubernetes/kubernetes/issues/111985
+      - name: Workaround upstream Kubernetes 1.25 bug
+        if: matrix.k8s == 'latest'
+        run: echo "KUBE_UNSTABLE_V1_25_DELETE=1" >> $GITHUB_ENV
 
       # Real CI work starts here
       - name: Build workspace
         run: cargo build
 
       # Run the equivalent of `just integration`
-      - name: Run all default features integration library tests
```

Comment:
Why was this name removed?

Reply:
Accident when refactoring from feature-flag to evar. Fixed now. Thanks!